src/DynamoCore/Library/XmlDocumentationExtensions.cs

```diff
@@ -101,34 +103,55 @@ private static string CleanUpDocString(this string text)
         }
 
         private static string GetMemberElement(FunctionDescriptor function,
-            string suffix, XDocument xml)
+            string property, XmlReader xml, string paramName = "")
         {
-            // Construct the entire function descriptor name, including CLR style names
-            string clrMemberName = GetMemberElementName(function);
+            var assemblyName = function.Assembly;
+            var fullyQualifiedName = GetMemberElementName(function);
+            if (!documentNodes.ContainsKey(fullyQualifiedName))
+                LoadDataFromXml(xml);
```

Comment:
Only if we can't find the `fullyQualifiedName` we need to call `DocumentationServices.GetForAssembly` as suggested above.